applications/vanilla/controllers/class.moderationcontroller.php

```diff
@@ -375,6 +375,10 @@ public function confirmDiscussionMoves($DiscussionID = null) {
         $DiscussionData = Gdn_DataSet::index($DiscussionData->resultArray(), ['DiscussionID']);
         foreach ($DiscussionData as $DiscussionID => $Discussion) {
             $Category = CategoryModel::categories($Discussion['CategoryID']);
+            if ($Category['Type'] === 'Reporting' && !array_key_exists('DiscussionType', $this->Data)) {
```

Comment:
Is this "Reporting" type from an addon or core Vanilla? If it's from an addon, we probably don't want to reference it like this in core.